symengine/lib/symengine_wrapper.pyx

```diff
@@ -387,6 +387,13 @@ def get_dict(*args):
     return D
 
 
+cdef tuple vec_basic_to_tuple(symengine.vec_basic vec):
```

Comment:
There's a overhead in copying. Passing a reference `symengine.vec_basic&` should fix that.